src/models/PostModel.ts

```diff
@@ -25,6 +25,9 @@ export class PostModel {
   @Column("text", { array: true })
   images: string[];
 
+  @Column("numeric")
+  created: number;
```

Comment:
is there a reason why we're using `number` type instead of `timestamptz`? https://www.postgresql.org/docs/current/datatype-datetime.html